### DISCUSSION TITLE
Timetable implementation

### DISCUSSION THREAD

**Discussion by rine77** [rine77/homeassistantedupage#26] [Show and tell]
... looks pretty much promising to me
![image](https://github.com/user-attachments/assets/1401ee90-fe89-48dc-926f-a1e837605fda)

**Comment by rine77** [comment]
... and the details are teacher, room and stuff
![image](https://github.com/user-attachments/assets/a25be019-41ec-45f6-8ec1-380b6081fa71)

**Comment by rine77** [comment]
it really gets in shape. We have multiple student support now.
Every student gets its own timetable calendar entry:

![image](https://github.com/user-attachments/assets/8a838fd2-b025-4efd-9ec1-7b0bef87059b)

**Reply by mgrfilipmarek** [reply]
wow, looks great. Btw, did you implemented also loading of grades for multiple students? (will be my case next year 😃 )

**Reply by rine77** [reply]
yes, it is in the next release.
Sensors will have names like
"John Doe Edupage Grades Physics"
And you can have concurrent students, so another sensor will be
"Jane Doe Edupage Grades Physics"

**Comment by brennyB** [comment]
THX @rine77 
Looks great (also for multiple students @mgrfilipmarek )

**Comment by rine77** [comment]
closed with #29
